### PR TITLE
Added Cart and Order Details (Both Web and mobile view)

### PR DESCRIPTION
**Pull request by mohakvni**

### CODE REVIEW

**Review by AnnLe4869 — CHANGES_REQUESTED** [review]
Besides all the comments I made, your code has inconsistent spacing. For example, do you want to have spaces before equal signs? In VSCode there is an extension called Prettier that can do formatting for you. https://prettier.io/docs/en/why-prettier.html

File: src/routes/checkout/Cart.js [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+    </Alert>
+  </>
+  )
+}
+
+const OutOfStock = () => {
+  return (
+    <>
+    <Alert key='danger' variant='danger' style={{
+            'position':'absolute',
```
[the remark this answers is not in the document]
Comment (mohakvni) [reply src/routes/checkout/Cart.js:127]: Gotcha, will do so

File: src/routes/checkout/Cart.js [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+      </Offcanvas>
+      
+      </main>
+    </div>
+  )
+}
+
+
+
+const closedstore = () => {
```
[the remark this answers is not in the document]
Comment (mohakvni) [reply src/routes/checkout/Cart.js:100]: Noted.

File: src/routes/checkout/Cart.module.css [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 21px;
+    display: flex;
+    align-items: center;
+    color: #7141FA;
+}
+
+.buttonnum {
```
[the remark this answers is not in the document]
Comment (mohakvni) [reply src/routes/checkout/Cart.module.css:60]: As far as I have tested, it should work on all the screens. If later on required, will change everything to percentage

**Comment by AnnLe4869** [comment]
Close and move the PR to another branch

**Pull request closed without merging**